src/get_target_data.R

```diff
@@ -78,35 +77,37 @@ get_target_data <- function(
   output_dirpath <- fs::path(base_hub_path, "target-data")
   fs::dir_create(output_dirpath)
 
-  raw_nhsn_data <- forecasttools::pull_nhsn(
+  nhsn_data <- forecasttools::pull_nhsn(
     api_endpoint = "https://data.cdc.gov/resource/mpgq-jmmr.json",
     columns = c("totalconfc19newadm"),
     start_date = first_full_weekending_date
-  )
-
-  output_file <- fs::path(output_dirpath, "time-series", ext = "parquet")
-  hubverse_format_nhsn_data <- raw_nhsn_data |>
+  ) |>
     dplyr::rename(
       observation = "totalconfc19newadm",
-      date = "weekendingdate",
-      state = "jurisdiction"
+      date = "weekendingdate"
     ) |>
     dplyr::mutate(
       date = as.Date(.data$date),
       observation = as.numeric(.data$observation),
-      state = stringr::str_replace(.data$state, "USA", "US")
+      jurisdiction = stringr::str_replace(.data$jurisdiction, "USA", "US")
     ) |>
-    dplyr::filter(!stringr::str_detect(.data$state, "Region")) |>
     dplyr::mutate(
-      location = forecasttools::us_location_recode(.data$state, "abbr", "code"),
+      location = forecasttools::us_location_recode(
+        .data$jurisdiction,
+        "abbr",
+        "code"
+      ),
       as_of = !!today,
       target = "wk inc covid hosp"
     ) |>
     dplyr::filter(!(location %in% !!excluded_locations))
 
-  hubverse_format_nhsn_data |>
+  hubverse_format_nhsn_data <- nhsn_data |> dplyr::select(-"jurisdiction")
+
+  nhsn_data |>
     dplyr::rename(
-      value = observation
+      value = observation,
+      state = jurisdiction
```

Comment:
```suggestion
      value = "observation",
      state = "jurisdiction"
```
good opportunity to improve compliance with https://potential-adventure-637em36.pages.github.io/r_coding_tips.html#for-tidy-selection-use-var-instead-of-var